gallery/package.json

```diff
@@ -60,6 +60,7 @@
     "@fortawesome/react-fontawesome": "=0.1.4",
     "@sonatype/react-shared-components": "link:../lib/dist",
     "classnames": "2.2.6",
+    "core-js": "3.6.5",
```

Comment:
Is this that library that always asks for money during npm install? I don't suppose there is something else we could use?